src/Sylius/Behat/spec/Context/Ui/CartContextSpec.php

```diff
@@ -89,6 +89,22 @@ function it_throws_not_equal_exception_if_promotion_total_is_incorrect(CartSumma
         $this->shouldThrow(NotEqualException::class)->during('myDiscountShouldBe', ['$50.00']);
     }
 
+    function it_ensures_there_is_no_grand_total_info_on_the_page(CartSummaryPage $cartSummaryPage)
+    {
+        $cartSummaryPage->open()->shouldBeCalled();
+        $cartSummaryPage->getGrandTotal()->willThrow(new ElementNotFoundException('"grand total" element is not present on the page'));
+
+        $this->cartTotalShouldNotBeDisplayed();
+    }
+
+    function it_ensures_there_is_no_shipping_fee_info_on_the_page(CartSummaryPage $cartSummaryPage)
+    {
+        $cartSummaryPage->open()->shouldBeCalled();
+        $cartSummaryPage->getShippingTotal()->willThrow(new ElementNotFoundException('"shipping total" element is not present on the page'));
+
+        $this->thereShouldBeNoShippingFee();
```

Comment:
We also need to check the case when the shipping fee will be found on the page. Same for the above.